extended/src/test/java/io/kubernetes/client/extended/kubectl/KubectlDeleteTest.java

```diff
@@ -207,6 +209,9 @@ void kubectlDelete() throws KubectlException, ApiException {
         KubectlDelete<V1Job> kubectlDelete = Kubectl.delete(V1Job.class);
         kubectlDelete.apiClient(apiClient);
         kubectlDelete.namespace("foo").name("bar");
+        DeleteOptions deleteOptions = new DeleteOptions();
+        deleteOptions.setPropagationPolicy("Foreground");
+        kubectlDelete.deleteOptions(deleteOptions);
```

Comment:
Can you add validation that this worked correctly to the URL validation below

Reply:
Added check below for URL validation and you were correct it was not passing deleteOptions, hence GenericKubernetesApi.java to pass it.